packages/react-core/src/components/Page/Page.tsx

```diff
@@ -125,7 +125,6 @@ export class Page extends React.Component<PageProps, PageState> {
     isManagedSidebar: false,
     isBreadcrumbWidthLimited: false,
     defaultManagedSidebarIsOpen: true,
-    onPageResize: (): void => null,
```

Comment:
Even though this technically unnecessary, we consider changes to default values breaking.  Could you please update the PR to change the logic at [conditional](https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/components/Page/Page.tsx#L155) instead.  
WE can open a separate issue to remove needed default in a breaking change release.

Reply:
@tlabaj we can't change the logic at the condition level. The condition itself is correct but the supplied values are not. The condition checks for the function's existence. And the function is always defined. So the condition is actually buggy and can never be false. That is what breaks the environment.

I've put all the details into the linked issue.

The only other option is adding a new flag that would just disable the observer. But again, looking at the condition logic, the existance of a prop is what should decide the outcome.

Reply:
One other alternative that would allow proper opting-out of the page resizing behavior without potentially causing breaking changes: we could update the type for `onPageResize` to be a union of the current type and `null` or something to that effect.

@Hyperkid123 do you have any thoughts regarding that potential approach? I know it isn't ideal, but I think it would be the simplest approach to getting you unblocked until our next breaking change release.

Reply:
Cool, I am on board with that. Let me update the PR then.

Reply:
@Hyperkid123 I opened issue #7861 to address the default value for our next major release.